src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt

```diff
@@ -41,10 +47,28 @@ object AutoExcuse : Module() {
         listener<PacketEvent.Receive> {
             if (mc.player == null || it.packet !is SPacketUpdateHealth) return@listener
             if (it.packet.health <= 0f && timer.tick(3L)) {
-                MessageSendHelper.sendServerMessage(getExcuse())
+                    MessageSendHelper.sendServerMessage(getExcuse())
             }
         }
     }
 
+    override fun onToggle() {
+        if (mode.value == Mode.READ_FROM_FILE) {
+            if (file.exists() && file.length() != 0L) {
+                excuses.clear()
+                file.forEachLine { excuses.add(it) }
+            } else {
+                MessageSendHelper.sendErrorMessage("$chatName File is empty / file does not exist! Disabling.")
+                MessageSendHelper.sendWarningMessage("$chatName To create custom excuses, create a file named 'excuses.txt' inside your '.minecraft' folder.")
```

Comment:
Change the message to not imply you need to make it, see #1453 (I'd normally leave it but we're changing it soon and I don't want to forget about some random string)